config.schema.json

```diff
@@ -126,123 +126,663 @@
               }
             },
             "command1": {
-              "title": "Custom Area 1",
+              "title": "Spot Area 1",
               "type": "string",
-              "description": "Enter a 'Spot Area' number or multiple numbers separated by a comma.",
+              "description": "Enter a spot area ID or multiple IDs separated by a comma.",
               "condition": {
-                "functionBody": "return (model?.devices?.[arrayIndices]?.deviceId?.length > 0 && !model?.devices?.[arrayIndices]?.ignoreDevice);"
+                "functionBody": "return (model?.devices?.[arrayIndices]?.deviceId?.length > 0 && !model?.devices?.[arrayIndices]?.ignoreDevice  && model?.devices?.[arrayIndices]?.command1);"
```

Comment:
please check this doesn't hide the field if nothing has been entered (eg for new users) or for an existing user that deletes the field)

Reply:
This is my intention - those fields  named "command1" to "command15" contain the custom area commands that are now meant to be migrated to the new predefined areas of type spot area. 

I kept those fields for backwards compatibility reasons. The migration needs to be done manually. Until then, everything works as before. I included a hint for the users (see line 864): 

```
Please move those entries to separate areas of type 'Spot Area' below to make use of the new note or custom area feature. IMPORTANT: You need to empty the old commands as they have priority for backwards compatibility reasons.
```

As soon as the user has migrated the old commands end emptied the fields they will disappear. New users are not meant to create the old commands anymore, but directly use the predefined areas instead.